src/xmipp/libraries/reconstruction/nma_alignment_vol.cpp

```diff
@@ -242,45 +246,53 @@ double ObjFunc_nma_alignment_vol::eval(Vector X, int *nerror) {
 
 	FileName fnRandom = global_nma_vol_prog->createDeformedPDB();
 	const char * randStr = fnRandom.c_str();
-	//FileName fnShiftsAngles = formatString("%s_angles_shifts.txt", randStr);
-	FileName fnShiftsAngles = fnRandom + "_angles_shifts.txt" ;
+	double retval=1e10;
+
+	FileName fnShiftsAngles = fnRandom + "_angles_shifts.txt";
 	const char * shifts_angles = fnShiftsAngles.c_str();
 
 	if (global_nma_vol_prog->alignVolumes){
-		if (global_nma_vol_prog->UseMissingWedgeMask){
-			runSystem("xmipp_volume_align",formatString("--i1 %s --i2 %s_deformedPDB.vol --frm --apply --store %s -v 0 --mask binary_file %s",
-					global_nma_vol_prog->currentVolName.c_str(),fnRandom.c_str(),shifts_angles,global_nma_vol_prog->fnMWmask.c_str()));
-			// Print to check in case of doubt
-			//std::cout << formatString("--i1 %s --i2 %s_deformedPDB.vol --frm --apply --store %s -v 0 --mask binary_file %s",
-			//		global_nma_vol_prog->currentVolName.c_str(),fnRandom.c_str(),shifts_angles,global_nma_vol_prog->fnMWmask.c_str());
+		if (global_nma_vol_prog->tilt0!=-90 || global_nma_vol_prog->tiltF!=90 ){
```

Comment:
Check that this works for other combinations of angles

Reply:
done.